android/build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
         targetSdkVersion = 34
         // We use NDK 23 which has both M1 support and is the side-by-side NDK version from AGP.
         ndkVersion = "21.4.7075529"
-        kotlinVersion = "1.9.0"
+        kotlinVersion = "1.9.23"
```

Comment:
why do we need to update this version?

Reply:
Earlier the release build was not getting built, so had to bump up patch version. But now its building it. So reverted it